frontend/src/stores/roles/state.ts

```diff
@@ -4,3 +4,11 @@ import { type Ref, ref } from 'vue'
 const roles: Ref<Role[]> = ref([])
 
 export { roles }
+
+export const removeRoleFromState = (roleId: number): void => { roles.value = roles.value.filter(role => role.id !== roleId); };
```

Comment:
This should be in the actions file, not the state. The state should contain just the data, without functions